lib/rubocop/cop/style/slicing_with_range.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks that arrays are sliced with endless ranges instead of
+      # `ary[start..-1]` on Ruby 2.6+
+      #
+      # @example
+      #   # bad
+      #   items[1..-1]
+      #
+      #   # good
+      #   items[1..]
+      class SlicingWithRange < Cop
+        extend TargetRubyVersion
+
+        minimum_target_ruby_version 2.6
+
+        MSG = 'When slicing array till its end, prefer endless range: '\
+              'ary[n..] instead of ary[n..-1].'
```

Comment:
Ideally here we should use the actual array name from the code, but that's not a big deal.

Reply:
Thought about it but decided against it: realistic case may look like  `chain.of(...).lot...of.operations(...)[calculating.the.first(...).index..-1]`, and trying to reproduce the whole statement in cop's message will blur the intention.